.chloggen/coralogixexporter-checkapi-26304.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
I don't think there's any user-facing change in this PR, you can remove this changelog.